api/database/factories/AssessmentResultFactory.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Enums\AssessmentDecision;
+use App\Enums\AssessmentResultJustification;
+use App\Enums\AssessmentResultType;
+use App\Enums\SkillDecisionLevel;
+use App\Models\AssessmentResult;
+use App\Models\AssessmentStep;
+use App\Models\PoolCandidate;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class AssessmentResultFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = AssessmentResult::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        $educationJustifications = [
```

Comment:
You're right that we need some way to categorize these.  I suggest that these arrays might logically live in the file with the enum definition.  Then you could grab them like `(array_column(AssessmentResultJustification::educationJustifications(), 'name')`.

Reply:
Oh, handy, didn't know you could add functions into there.
https://github.com/GCTC-NTGC/gc-digital-talent/pull/8302/commits/d54acdf537529543fc17918c8681b21eacd5c02c